gulp/tasks/content.js

```diff
@@ -0,0 +1,46 @@
+var path = require('path');
+var gulp = require('gulp');
+var clone = require('gulp-clone');
+var buffer = require('vinyl-buffer');
+var mergeStream = require('merge-stream');
+var generateContentAndTemplates = require('../util/content').generateContentAndTemplates;
+var through = require('through2');
+
+gulp.task('content', function() {
+  var base, contentJSON, languagesJSON, matchHTML, templates, templateStreams;
+
+  base = path.resolve(__dirname, '..', '..');
+  contentJSON = require('../../source/content.js');
+  languagesJSON = require('../../source/languages.js');
+  matchHTML = /\.html$/;
+  templates = {};
+  templateStreams = mergeStream();
```

Comment:
Ok, I'll take a look then. There's an example below that is particularly bad and takes up 2 lines. :S

Re: style guide: I'd move to just follow https://github.com/jshttp/style-guide/blob/master/javascript.md

(I compiled most of it, so I __am__ biased.)